src/main/scala/com/lucidworks/spark/util/ConfigurationConstants.scala

```diff
@@ -27,5 +28,15 @@ object ConfigurationConstants {
   val SAMPLE_SEED: String = "sample_seed"
   val SAMPLE_PCT: String = "sample_pct"
 
+  // Time series partitioning params
+
+  val PARTITION_BY:String="partition_by"
+  val TIME_STAMP_FIELD_NAME:String="time_stamp_field_name"
+  val TIME_PERIOD:String="time_period"
+  val DATETIME_PATTERN:String="datetime_pattern"
+  val TIMEZONE_ID:String="timezone_Id"
```

Comment:
don't use uppercase in option names, should be `timezone_id` ... fix the readme too